packages/labs/analyzer/src/test/analyzer_test.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {test} from 'uvu';
+import * as assert from 'uvu/assert';
+import ts from 'typescript';
+import * as path from 'path';
+
+import {Analyzer} from '../lib/analyzer.js';
+
+test('Basic elements - isLitElement', () => {
+  const basicElementsPackagePath = new URL(
+    '../test-files/basic-elements',
+    import.meta.url
+  ).pathname;
```

Comment:
Won't this always be a `/` delimited path? So it won't work on Windows. I think you want:

```ts
import * as pathlib from 'path';
import {fileURLToPath} from 'url';

const __filename = fileURLToPath(import.meta.url);
const __dirname = pathlib.dirname(__filename);
const basicElementsPackagePath = pathlib.resolve(__dirname, '..', 'test-files', 'basic-elements');
```

Reply:
I guess this would work too, a bit simpler:

```ts
import {fileURLToPath} from 'url';

fileURLToPath(new URL(
  '../test-files/basic-elements',
  import.meta.url
).pathname));
```